spec/active_operation/pipeline_spec.rb

```diff
@@ -111,6 +125,26 @@ def execute
     it { is_expected.to utilize_operations(string_generator, string_capitalizer) }
   end
 
+  context(
+    "when composed of two operations, one that generates a string and one that lowercases strings by receiving its " \
+    "input as a required property"
+  ) do
+    subject(:composed_operation) do
+      operations = [string_generator, string_lowercaser_by_property]
+
+      Class.new(described_class) do
+        use operations.first
+        use operations.last, text: -> { previous_output }
```

Comment:
This contradicts the vision I have for pipelines. If you want Operation A to supply the keyword arguments for the subsequent Operation B, then I expect A to return a hash compatible with the properties of B.